packages/integration-tests/joist-codegen.json

```diff
@@ -13,6 +15,14 @@
     "Book": { "tag": "b" },
     "BookAdvance": { "tag": "ba" },
     "BookReview": { "fields": { "isPublic": { "derived": "async" } }, "tag": "br" },
+    "Critic": {
+      "fields": {
+        "ignoreFavouriteBook": { "ignore": true },
+        "ignoreWorstBook": { "ignore": true },
```

Comment:
Fwiw having `ignore` in the column name itself threw me off ever so slightly in reading some of the test, kinda like the `shouldNotIgnore` field name in the other test.

Reply:
Yea, i could see that. I felt like it was helpful in the migration though to make it clear that you won't be seeing it in the codegen output? I will update the `shouldNotIgnore` field name to something else.